rascaline/src/calculators/neighbor_list.rs

```diff
@@ -37,6 +37,8 @@ pub struct NeighborList {
     /// `i-j` and once as `j-i`), or a half neighbor list (each pair only
     /// appears once)
     pub full_neighbor_list: bool,
+    /// Should we compute self-neighbor terms
+    pub self_terms: bool,
```

Comment:
Other parts of the code use `self_pairs`, could you change this to be the same? https://github.com/Luthaf/rascaline/blob/f180ada9edbcc441f7543153d6f78fe05f5cb490/rascaline/src/labels/samples/atom_centered.rs#L23

Reply:
Done!